### ISSUE TITLE
Time-bound vs endless campaign functionality

### ISSUE THREAD

**Issue by mfc** [izens-net/picket#24]
Given endless campaigns impact user functionality more, there should be reasons why someone would want to do a time-bound campaign, and also have increased responsibility/accountability for endless blocking campaigns.

For time-bound campaigns, the campaign creator could have pre-notification to users (warn for *x* days prior to campaign for blocking actions).

For endless campaigns, maybe the policy file needs to be updated by its maintainer once a year (or similar) in order to stay active -- otherwise unmaintained campaigns may result in user actions that are not actually real-world relevant. This may require izens signing policy files or some other authentication or certification of policy files (and creation/update dates). Not sure we really want to go down that route...

**Comment by anikarni** [comment]
Should we split these into two different issues, one which implements the notification and one which implements certification/signing or whatever that would look like? I think the latter requires more discussion, whereas the first can already be picked up by someone soon.

**Comment by mfc** [comment]
yes that sounds good! agreed the second requires additional discussion.
